alembic/versions/a9363331e323_clean_h2o_tests_with_removed_shuffle_.py

```diff
@@ -0,0 +1,35 @@
+"""Clean h2o tests with removed shuffle param
+
+Revision ID: a9363331e323
+Revises: c38b9d85915e
+Create Date: 2023-01-03 19:56:22.838577
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a9363331e323'
```

Comment:
Likely need to coordinate merges with https://github.com/coiled/coiled-runtime/pull/632 and https://github.com/coiled/coiled-runtime/pull/643

Instead we can also drop the migrations from both PRs and do both changes in one alembic migration